src/components/ShowsList.js

```diff
@@ -0,0 +1,150 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  fetchShows, selectAllShows, selectShowsStatus, selectShowsError,
+} from '../redux/shows_list/listShowsSlice';
+import { WebHeader, MobileHomeHeader } from './navigation/Header';
+
+const ShowList = () => {
+  const dispatch = useDispatch();
+  const shows = useSelector(selectAllShows);
+  const status = useSelector(selectShowsStatus);
+  const error = useSelector(selectShowsError);
+  const totalCount = shows.length;
+  const endedShows = shows.filter((show) => show.status === 'Ended');
+  const endedShowsCount = endedShows.length;
+  const runningShows = shows.filter((show) => show.status === 'Running');
+  const runningShowsCount = runningShows.length;
+  const [searchInput, setSearchInput] = useState('');
+
+  const filteredShows = shows.filter((show) => show.name.toLowerCase().includes(searchInput.toLowerCase()));
+
+  useEffect(() => {
+    dispatch(fetchShows());
+  }, [dispatch]);
+
+  if (status === 'loading') {
+    return <div>Loading...</div>;
+  }
+
+  if (status === 'failed') {
+    return (
+      <div>
+        Error:
+        {error}
+      </div>
+    );
+  }
+
+  return (
+    <div className="App">
+
+      <div className="webHeader">
+        <WebHeader />
+      </div>
+      <div className="mobileHeader">
+        <MobileHomeHeader />
+      </div>
+      <input
+        type="text"
+        placeholder="Search shows..."
+        className="webSearch"
+        onChange={(e) => setSearchInput(e.target.value)}
+      />
```

Comment:
- [ ] Good work on adding the search feature! 👏 To enhance it, consider making the search bar more appealing and user-friendly:

1. **Color**: Use a color that makes the search bar stand out.
2. **Text Size**: Make the text easy to read.
3. **Placeholder**: Add clear text like "Search here..."
4. **Search Icon**: Include a magnifying glass icon inside the bar.

These tweaks will make the search bar look and feel better. See below for more details on your current design:
<img width="396" alt="Screenshot 2023-09-29 at 14 49 15" src="https://github.com/paulshaun-kdtk/React_Capstone_Metrics/assets/107192076/266b8b7f-051e-4713-b875-3716d8c4e362">